docs/blueprints/002-extension-registry.md

```diff
@@ -0,0 +1,526 @@
+              url: http://mlflow
+              type: internal
+            - name: ingress
+              url: http://mlflow.10.110.120.130.nip.io
+              type: external
+              default: True
+        - name: mlflow-store
+          service: s3
+          description: MLFlow minio S3 storage back-end
+          profiles:
+            - default-s3-account
+          endpoints:
+            - name: cluster
+              url: http://mlflow-minio:9000
+              type: internal
+            - name: ingress
+              url: http://mlflow.10.110.120.130.nip.io
+              type: external
+              default: True
+      profiles:
+        - name: default-s3-account
+          scope: global
+          configuration:
+            - name: AWS_ACCESS_KEY_ID
+              value: 24oT0SfbJPEu6kUbUKsH
+            - name: AWS_SECRET_ACCESS_KEY
+              value: cMGiZff8KqS5xWQ4eagRujh1tDcbQyRP0bEJSBOf
+  ```
+
+2. example showing that the minio sub-service from the previous MLFlow instance can also be registered as a generic minio/S3 service, although this is not recommended, because even though the s3 service can be consumed independently of the parent product instance, the way that data is organized and stored in the s3 back-end is specific to MLFlow and should be discoverable as such:
+
+  ```yaml
+  name: minio
+  product: minio
+  description: Minio S3 storage service
+  instances:
+    - name: mlflow-storage
+      version: "4.1.3"
+      location: local
+      services:
+        - name: s3
+          service: s3
+          description: MLFlow minio S3 storage back-end
+          profiles:
+            - default
+          endpoints:
+            - name: cluster
+              url: http://mlflow-minio:9000
+              type: internal
+            - name: ingress
+              url: http://mlflow.10.110.120.130.nip.io
+              type: external
+              default: True
+      profiles:
+        - name: default
+          scope: global
+          configuration:
+            - name: AWS_ACCESS_KEY_ID
+              value: 24oT0SfbJPEu6kUbUKsH
+            - name: AWS_SECRET_ACCESS_KEY
+              value: cMGiZff8KqS5xWQ4eagRujh1tDcbQyRP0bEJSBOf
+  ```
+
+3. example of an extension record for a third party gitea instance hosted in a location other than FuseML where each FuseML project is associated with a Gitea user
+
+  ```yaml
+  name: gitea
+  product: gitea
+  description: Gitea version control server
+  instances:
+    - name: gitea-devel-rd
+      version: "1.14.3"
+      location: remote
+      services:
+        - name: git+https
+          service: git+https
+          description: Gitea git/https API
+          endpoints:
+            - name: public
+              url: https://mlflow-minio:9000
+              type: external
+              default: True
+      profiles:
+        - name: default
+          scope: global
+          configuration:
+            - name: username
+              value: fuseml
```

Comment:
I know it's the example, but could you put admin user name here instead? It's closer to the way we're using gitea now...

Reply:
ok